src/rdf4cpp/Node.hpp

```diff
@@ -1,27 +1,31 @@
 #ifndef RDF4CPP_NODE_HPP
 #define RDF4CPP_NODE_HPP
 
-#include <rdf4cpp/rdf/storage/node/NodeStorage.hpp>
-#include <rdf4cpp/rdf/util/TriBool.hpp>
-#include <rdf4cpp/rdf/writer/BufWriter.hpp>
+#include <rdf4cpp/storage/NodeStorage.hpp>
+#include <rdf4cpp/storage/identifier/NodeBackendHandle.hpp>
+#include "TriBool.hpp"
```

Comment:
full path

Reply:
It is really annoying that clion always does this, and hides includes..